From a1093f67601e8dbe55cf33460f1c6262981df3bf Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 27 Mar 2013 13:24:25 +0800
Subject: [PATCH 55/58] ENGR00256086 Connectivity:let it deprecated for
 connectivity

Delete Android.mk to let it deprecated. Move it to device/fsl-proprietary.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 firmware/ar3k/Android.mk  | 32 -----------------------------
 firmware/ath6k/Android.mk | 51 -----------------------------------------------
 2 files changed, 83 deletions(-)
 delete mode 100644 firmware/ar3k/Android.mk
 delete mode 100644 firmware/ath6k/Android.mk

diff --git a/firmware/ar3k/Android.mk b/firmware/ar3k/Android.mk
deleted file mode 100644
index 0d61612..0000000
--- a/firmware/ar3k/Android.mk
+++ /dev/null
@@ -1,32 +0,0 @@
-ifeq ($(BOARD_HAVE_BLUETOOTH),true)
-ifeq ($(BOARD_USE_AR3K_BLUETOOTH),true)
-LOCAL_PATH := $(call my-dir)
-AR_FW_PATH := 30101
-INSTALL_PATH := $(TARGET_OUT)/lib/firmware/ar3k/$(AR_FW_PATH)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := ar3kbdaddr.pst
-LOCAL_MODULE_TAGS := eng
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := PS_ASIC.pst
-LOCAL_MODULE_TAGS := eng
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := RamPatch.txt
-LOCAL_MODULE_TAGS := eng
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-endif
-endif
diff --git a/firmware/ath6k/Android.mk b/firmware/ath6k/Android.mk
deleted file mode 100644
index be64b25..0000000
--- a/firmware/ath6k/Android.mk
+++ /dev/null
@@ -1,51 +0,0 @@
-
-ifeq ($(BOARD_SOC_CLASS), IMX5X)
-ifeq ($(BOARD_HAVE_WIFI), true)
-
-LOCAL_PATH := $(call my-dir)
-AR_FW_PATH := AR6102
-INSTALL_PATH := $(TARGET_OUT)/lib/firmware/ath6k/$(AR_FW_PATH)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := athwlan.bin.z77
-LOCAL_MODULE_TAGS := user
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_MODULE_TAGS := eng
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := data.patch.hw2_0.bin
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_MODULE_TAGS := eng
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := eeprom.data
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_MODULE_TAGS := eng
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := eeprom.bin
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_MODULE_TAGS := eng
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-LOCAL_MODULE := calData_ar6102_15dBm.bin
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(INSTALL_PATH)
-LOCAL_MODULE_TAGS := eng
-LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-endif
-endif
-- 
1.8.0

